### DISCUSSION TITLE
Finding Errors while calling the @paddle-js-models/ocr in angular and node.js

### DISCUSSION THREAD

**Discussion by anixsam** [PaddlePaddle/PaddleOCR#8240] [Q&A]
Its showing Break Changing Error for the Angular and ESM library error for the node js

**Comment by LDOUBLEV** [comment]
Can you provide your code to reproduce the problem?

**Comment by anixsam** [comment]
I’ve followed the sample code provided in the example and getting breaking
changes error in angular
